wdtk-client/src/main/java/org/wikidata/wdtk/client/RdfSerializationAction.java

```diff
@@ -251,4 +254,21 @@ private void printHelp() {
 					+ TASK_HELP.get(supportedTask), 75, "\n   ", true));
 		}
 	}
+
+	@Override
+	public String getReport() {
+		String message = "Finished serialization of "
+				+ this.serializer.getTripleCount() + " RDF triples in file "
+				+ this.outputDestination;
+		return message;
+	}
+
+	@Override
+	public String getActionName() {
```

Comment:
Yes, that would be an acceptable solution too. I think, however, that such additional get methods are not necessary here: I don't see why one should return something else than the class name. It seems like a good default, and subclasses could still overwrite it by calling the setter in their constructor with another string.